processing/src/main/java/org/apache/carbondata/processing/loading/sort/unsafe/UnsafeSortDataRows.java

```diff
@@ -127,8 +127,11 @@ public UnsafeSortDataRows(SortParameters parameters,
   /**
    * This method will be used to initialize
    */
-  public void initialize() throws MemoryException, CarbonSortKeyAndGroupByException {
+  public void initialize() throws MemoryException {
     this.rowPage = createUnsafeRowPage();
+    if (this.rowPage == null) {
```

Comment:
No need to check null case here if it initialize fails throw exception no need to check for null

Reply:
ok. removed it